### PR TITLE
Sso documentation

### PR DESCRIPTION
**Pull request by vaimdevs**
SSO documentation for public

### CODE REVIEW

**Comment by lmeyerov** [comment]
@dess890 can you review, and if any grammar etc, fix on the branch?

File: docs/OIDC_provider_setup.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+## AUTH0 SETUP
+
+### AUTH0 OIDC SETUP
+
+1. After signing an account for Auth0, select “Company” as the account type so you can limit the person who can log in to this organization. Fill in the company name and select size for company.	
+![image](https://user-images.githubusercontent.com/93269128/159425323-b0b2d57c-bfed-4887-aebd-7a4e54919d20.png)
+2. You can change the domain name and choose the country you want. Change of country will change the host url. 
+![image](https://user-images.githubusercontent.com/93269128/159425368-8f7b0fa1-9106-4c51-a4c6-eed5e7586da2.png)
+3. Click the “Application” in the Application section to go to the application page. Click “Create Application” to create a new application.
```
Comment (dess890) [inline docs/OIDC_provider_setup.md]: Under the application panel, click “Create Application” to create a new application ***

File: docs/OIDC_provider_setup.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+1. After signing an account for Auth0, select “Company” as the account type so you can limit the person who can log in to this organization. Fill in the company name and select size for company.	
+![image](https://user-images.githubusercontent.com/93269128/159425323-b0b2d57c-bfed-4887-aebd-7a4e54919d20.png)
+2. You can change the domain name and choose the country you want. Change of country will change the host url. 
+![image](https://user-images.githubusercontent.com/93269128/159425368-8f7b0fa1-9106-4c51-a4c6-eed5e7586da2.png)
+3. Click the “Application” in the Application section to go to the application page. Click “Create Application” to create a new application. 
+![image](https://user-images.githubusercontent.com/93269128/159425549-6be44aa8-b678-4ab9-be96-aa7536760c33.png)
+4. Name your application and select “Single Page Web Applications”. 
+![image](https://user-images.githubusercontent.com/93269128/200610755-2c4f973c-1125-4c39-9663-0bb7996ad7ec.png)
+5. Go to “Settings” to get “Domain”, “Client ID” and “Client Secret”.  We have to use these fields to register an Organization SSO ID Provider in graphistry.
```
Comment (dess890) [inline docs/OIDC_provider_setup.md]: Capitalize G in Graphistry

File: docs/OIDC_provider_setup.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+1. Go to ”User Management” and select the user to go to the user page. Click “Create User” to create a new user. 
+![image](https://user-images.githubusercontent.com/93269128/159425732-0a9052e1-b1a1-40ad-94af-4d82c3a88110.png)
+2. Fill in all of the fields and click “Create” to create a new user. 
+![image](https://user-images.githubusercontent.com/93269128/159425784-ef15759e-bb20-415d-b712-d65ac689dbe3.png)
+
+### AUTH0 ORGANIZATION SETUP
+
+1. Go to “Organizations” and click “Create Organization” to create a new organization. 	
+![image](https://user-images.githubusercontent.com/93269128/159425859-295b9e3b-1c59-4474-9697-07da4db63371.png)
+2. Name your organization and a name displayed.
```
Comment (dess890) [inline docs/OIDC_provider_setup.md]: Name your organization and set your displayed name. ***

File: docs/OIDC_provider_setup.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+![image](https://user-images.githubusercontent.com/93269128/159425894-3a75a592-df49-459b-a68e-0794aab9dd9e.png)
+3. Please record this “Organization ID” because this ID has to use when create org sso id provider.
```
Comment (dess890) [inline docs/OIDC_provider_setup.md]: Record your “Organization ID” which you'll need when you create your org sso id provider. **

File: docs/OIDC_provider_setup.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+### AUTH0 ORGANIZATION SETUP
+
+1. Go to “Organizations” and click “Create Organization” to create a new organization. 	
+![image](https://user-images.githubusercontent.com/93269128/159425859-295b9e3b-1c59-4474-9697-07da4db63371.png)
+2. Name your organization and a name displayed.
+![image](https://user-images.githubusercontent.com/93269128/159425894-3a75a592-df49-459b-a68e-0794aab9dd9e.png)
+3. Please record this “Organization ID” because this ID has to use when create org sso id provider. 	
+![image](https://user-images.githubusercontent.com/93269128/159425928-da97e42b-cfb3-457e-ac29-34f7a67fff92.png)
+4. You can change these fields to adjust thein UI the login page. Save changes if you change anything.
```
Comment (dess890) [inline docs/OIDC_provider_setup.md]: You can change these fields to adjust the UI the login page. Make sure to save your changes. ***

File: docs/OIDC_provider_setup.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+![image](https://user-images.githubusercontent.com/93269128/200621154-9b8e4e11-ac1a-44f8-afc2-19b9f8a6ce49.png)
+8. Select "Username-Password-Authentication" and click "Enable Connection".
+![image](https://user-images.githubusercontent.com/93269128/200621448-e8314aff-21a8-4473-af76-5483c4695d6d.png)
+
+
+## KEYCLOAK SETUP
+
+### KEYCLOAK OIDC SERVER SETUP
+
+1. Login to the keycloak admin console. Default username is “admin” and passwords is “graphistry”.
```
Comment (dess890) [inline docs/OIDC_provider_setup.md]: Login to the keycloak admin console. Default username is “admin” and password is “graphistry”. **

**Review by dess890 — CHANGES_REQUESTED** [review]

**Pull request merged**